instrumentation/spring/spring-batch-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/batch/chunk/TracingChunkExecutionListener.java

```diff
@@ -27,32 +31,40 @@ public TracingChunkExecutionListener(
 
   @Override
   public void beforeChunk(ChunkContext chunkContext) {
-    Context context = tracer().startSpan(chunkContext, builderClass);
+    Context parentContext = shouldCreateRootSpanForChunk() ? Context.root() : currentContext();
```

Comment:
Please use the Java 8 bridge for `root()` too:
```suggestion
    Context parentContext = shouldCreateRootSpanForChunk() ? rootContext() : currentContext();
```